.github/workflows/trivy.yml

```
@@ -0,0 +1,57 @@
+name: Trivy Scanner
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 8 * * *" # runs at 8 AM UTC, 4 AM EST.
```

```suggestion
    - cron: "0 8 * * *" # runs at 8 AM UTC, 3 AM EST.
```
Any reason to not run this at 1AM like the nightly test? Is the worry that it will compete for GitHub runner resources?

---

Yes, I think we should stagger the Github actions.

---

Why do you think we should do this? What improvements will we see if this is changed?

---

I think we should stagger Github actions to avoid resource contention and to make debugging easier.